src/VoxelRayTrace.php

```diff
@@ -46,11 +48,21 @@ public static function inDirection(Vector3 $start, Vector3 $directionVector, flo
 	 * Performs a ray trace between the start and end coordinates. This returns a Generator which yields Vector3s
 	 * containing the coordinates of voxels it passes through.
 	 *
+	 * The first Vector3 is `$start->floor()`.
+	 * Every subsequent Vector3 has a taxicab distance of exactly 1 from the previous Vector3;
+	 * if the ray crosses the intersection of multiple axis boundaries directly,
+	 * the algorithm prefers crossing the boundaries in the order `Z -> Y -> X`.
+	 *
+	 * If `$end` is on an axis boundary, the final Vector3 may or may not cross that boundary.
```

Comment:
This is fuzzier than I would like. How did you come up with this?

Reply:
by running the function with many test inputs.

Reply:
do you have any sample data that show this inconsistency?